sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,14 +1,14 @@
 # Release History
 
-## 1.4.0-beta.4 (Unreleased)
+## 1.4.0 (2023-02-07)
 
 ### Features Added
```

Comment:
I like we are emphasizing the key features added for the GA release.

Reply:
Thank you, I also prefer this over the "Changes from Beta are now generally available", which would be most of the time the GA changelog, given that we do release as Beta first most of the time.

Yet this is not the full verbatim details concatenated from beta releases - some we do summarize in shorter sentences, or not mention at all (i.e. bug fixes for a bug introcuced in one beta, fixed in subsequent beta).